press/article.md

```diff
@@ -0,0 +1,92 @@
+# Introduction
+
+In the recent react project we needed an environment to create, style and test our components without having a server-side backend code. We decided to use storybook that allows us to develop UI components outside the app and allow other people in your team to work on them. 
+
+We really enjoy using React Storybook because:
+1. Completely isolate the environment for your components
+2. HMR — hot module replacement
+3. Support for CSS
+4. Multiple add-ons like: 
+    + [Accessibility](https://github.com/joscha/storybook-addon-i18n-tools)
+    + [CSS regression](https://github.com/tsuyoshiwada/storybook-chrome-screenshot)
+    + [Snapshot testing of React components](https://github.com/storybooks/storybook/tree/master/addons/storyshots)
+    + [Knobs allow you to edit React props dynamically](https://github.com/storybooks/storybook/tree/master/addons/knobs)
+
+After using storybook in our project for a while,  we needed a way to display the CSS code samples that the component used inside the storybook panel, after doing our fair share of research into existing solutions we've came up empty, so we decided to create our own add-on called [storybook-addon-code](https://github.com/tuchk4/storybook-readme). 
```

Comment:
Phrase it too long. end it after 'inside the storybook panel' and start a new one: 'After doing ....'.

Reply:
Solved